tests/preview_changed_file/deleted_in_working.fish

```diff
@@ -1,7 +1,5 @@
 mock git diff "echo \$argv"
-set file dir/file.txt
+set output (_fzf_preview_changed_file " D out.log")
 
-set output (_fzf_preview_changed_file " D $file")
-
-contains -- "| Unstaged |" $output && not contains "| Staged |" $output
```

Comment:
add -- to the second invocation of `contains` to protect it from any output starting with `-`